harness/determined/cli/job.py

```diff
@@ -100,6 +87,83 @@ def update(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def process_updates(args: Namespace) -> None:
+    session = setup_session(args)
+    for arg in args.operation:
+        inputs = {"session": session}
+        values = arg.split(".")
+        if len(values) != 2:
+            raise ValueError(
+                f"Invalid operation specified for job {values[0]}. "
+                f"Please ensure they are formatted as <jobID>.<operation>=<value>."
+            )
+        inputs["job_id"] = values[0]
+        operation = values[1].split("=")
+        if len(operation) != 2:
+            raise ValueError(
+                f"Operation for job {values[0]} has invalid format. "
+                f"Please ensure it is formatted as <jobID>.<operation>=<value>."
+            )
+        inputs[operation[0]] = operation[1]
+        replace_keys(inputs)
+        _single_update(**inputs)
+
+
+def _single_update(
+    job_id: str,
+    session: Session,
+    priority: str = None,
+    weight: str = None,
+    resource_pool: str = None,
+    behind_of: str = None,
+    ahead_of: str = None,
+) -> None:
+    update = bindings.v1QueueControl(
+        jobId=job_id,
+        priority=priority,
+        weight=int(weight),
+        resourcePool=resource_pool,
+        behindOf=behind_of,
+        aheadOf=ahead_of,
+    )
+    bindings.post_UpdateJobQueue(session, body=bindings.v1UpdateJobQueueRequest([update]))
+    return
+
+
+def is_priority_rm(config: dict) -> bool:
+    try:
+        if config["resource_manager"]["scheduler"]["type"] == "fair_share":
+            return False
+    except KeyError:
+        pass
+    return True
+
+
+def check_is_priority(config: dict, resource_pool: str) -> bool:
+    try:
+        for pool in config["resource_pools"]:
+            if pool["pool_name"] == resource_pool and pool["scheduler"]["type"] == "fair_share":
+                return False
+        return is_priority_rm(config)
+
+    except (KeyError, TypeError):
+        return is_priority_rm(config)
+    return True
+
+
+def replace_keys(operations: dict) -> None:
```

Comment:
maybe
```suggestion
def validate_operation_args(operations: dict) -> None:
```

and have this function additionally check for validity of operation, printing a human-readable error when it's not right?